docs/tutorials/multimodal/object_detection/detection_eval_yolov3_coco.md

```diff
@@ -0,0 +1,87 @@
+# AutoMM Detection - Evaluate Pretrained YOLOv3 on COCO Format Dataset
+:label:`sec_automm_detection_eval_yolov3_coco`
+
+In this section, our goal is to evaluate YOLOv3 model on VOC2007 dataset in COCO format. We start with yolov3 because it's extremely fast and accurate, and is a good choice to deploy with strict time and computational restrictions.
```

Comment:
According to the json name, I think we are still evaluating COCO2017, not VOC2007 in COCO format, right?

Reply:
Just changed the typo here to COCO2017. Thanks!